### ISSUE TITLE
Feature request - ability to open container app view for functon app in env apps list

### ISSUE THREAD

**Issue by eduards-vavere** [Azure/azure-functions-on-container-apps#41]
In container app env apps list the function app is shown, but there is no button to open the container app for this function app. Workaround - have to search in all resources by name.

**Comment by raorugan** [comment]
@eduards-vavere  - may i know why you want to access the underlying container app resource ?

**Comment by eduards-vavere** [comment]
Of course! To do configuration on it like any of my other container apps. Ingress for example.

**Comment by raorugan** [comment]
Adding this issue link as similar ask is being discussed - https://github.com/Azure/azure-functions-on-container-apps/issues/27

**Comment by eduards-vavere** [comment]
Your reply is a bit of a surprise for me! I expected the function app to be just a regular container app I can manage as any other container app1 (and it is possible, because the container app resources exists!) For example:
- setting the revision mode to multiple for blue/green deployments.
- setting cpu/memory KEDA scalers
- using container app secrets
- setup health probes
- setup resiliency for dapr bindings (use dapr binding for function app calls to other container apps)

So you are saying I should not even be using the container app resources (with the same name as) function app?

**Comment by raorugan** [comment]
Sure!! would like to discuss more on this , on the scenario and requirements  .. Please drop a note to raorugan@microsoft.com

**Comment by eduards-vavere** [comment]
@raorugan sure, we can have a call, you can message me
